clippy_dev/src/new_lint.rs

```diff
@@ -98,7 +98,7 @@ fn create_test(lint: &LintData<'_>) -> io::Result<()> {
         fs::create_dir(&test_dir)?;
 
         create_project_layout(lint.name, &test_dir, "fail", "Content that triggers the lint goes here")?;
-        create_project_layout(lint.name, &test_dir, "pass", "This file should not trigger the lint")?;
+        create_project_layout(lint.name, test_dir, "pass", "This file should not trigger the lint")?;
```

Comment:
This suggestion is not particularly great. It raises the question of why the first call borrows `test_dir`, but the second call doesn't. Reusing `test_dir` afterwards on a later change is now extra work as well.